### PR TITLE
chore(scripts): Adjust design assets import script

### PR DESCRIPTION
**Pull request by paroxyzm**
This PR updates assets-import script.
It will take assets from `_Deprecated` directory instead of the root directory

### CODE REVIEW

File: scripts/asset-import/cli.js [hunk 1]
```diff
@@ -16,6 +16,10 @@ const { argv } = yargs
         alias: 'b',
         description: 'The branch or tag to use',
     })
+    .option('assetDirectory', {
+        alias: 'a',
+        description: 'Directory from which to take assets',
+    })
     .option('working-dir', {
```
Comment (tjuanitas) [inline scripts/asset-import/cli.js:23]: nit: the other options are kebab cased

```suggestion
    .option('asset-dir', {
        alias: 'a',
        description: 'Directory from which to take assets',
    })
    .option('working-dir', {
```

File: scripts/asset-import/cli.js [hunk 2]
```diff
@@ -33,7 +37,9 @@ const { argv } = yargs
 
 (async function cli() {
     try {
-        await importer(pick(argv, ['repository', 'branch', 'workingDir', 'destinationDir', 'verbose']));
+        await importer(
+            pick(argv, ['repository', 'branch', 'workingDir', 'destinationDir', 'verbose', 'assetDirectory']),
```
Comment (tjuanitas) [inline scripts/asset-import/cli.js:41]: does the order matter here?

Comment (tjuanitas) [reply scripts/asset-import/cli.js:41]: probably not i guess

File: scripts/asset-import/src/defaultConfig.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
+    assetDirectory: '_Deprecated',
     branch: 'master',
     workingDir: '/tmp',
     destinationDir: 'src',
```
Comment (tjuanitas) [inline scripts/asset-import/src/defaultConfig.js:5]: nit: the other directory names are shortened to `Dir`

```suggestion
    assetDir: '_Deprecated',
    branch: 'master',
    workingDir: '/tmp',
    destinationDir: 'src',
```

File: scripts/asset-import/src/import.js [hunk 4]
```diff
@@ -91,6 +91,7 @@ module.exports = async cliOptions => {
 
     const git = new SimpleGit(config.workingDir);
     const sourcePath = `${config.workingDir}/${uniqid('design-assets-')}`;
+    const assetDirectory = `${sourcePath}/${config.assetDirectory}`;
```
Comment (tjuanitas) [inline scripts/asset-import/src/import.js:94]: could we call this `assetPath`?

**Review by tjuanitas — COMMENTED** [review]
thanks for taking a look and updating this!

File: scripts/asset-import/cli.js (hunk 2 above)
Comment (paroxyzm) [reply scripts/asset-import/cli.js:41]: no

File: scripts/asset-import/src/import.js (hunk 4 above)
Comment (paroxyzm) [reply scripts/asset-import/src/import.js]: Yes, changed

**Review by tjuanitas — APPROVED** [review]

**Review by tjuanitas — APPROVED** [review]

**Pull request merged**